.github/actions/release/action.yml

```diff
@@ -193,6 +193,8 @@ runs:
         JIRA_RELEASE_NAME=""
         JIRA_PROJECT_ID="${{ inputs.jira_project_id }}"
         JIRA_RELEASE_RELEASED="false"
+        JIRA_RELEASE_ID="$(git log |grep -E "Centreon\ next.*\#[0-9]{5,}\#\)" |grep -o -P "(?<=#)[0-9]{5,}(?=#)" |head -n 1)"
```

Comment:
seems not used

Reply:
I think this was due to missing merge os sync release 22.10, it is now used at L212